torch/_dynamo/variables/builtin.py

```diff
@@ -1677,11 +1677,21 @@ def _unimplemented():
         ):
             return ConstantVariable.create(op(left.value, right.value))
 
-        if (
-            (isinstance(left, StreamVariable) and isinstance(right, StreamVariable))
-            or (isinstance(left, EventVariable) and isinstance(right, EventVariable))
-        ) and op is operator.eq:
-            return ConstantVariable(op(left.value, right.value))
+        if isinstance(left, (StreamVariable, EventVariable)) or isinstance(
+            right, (StreamVariable, EventVariable)
+        ):
+            if (
+                (isinstance(left, StreamVariable) and isinstance(right, StreamVariable))
+                or (
+                    isinstance(left, EventVariable) and isinstance(right, EventVariable)
+                )
+            ) and op is operator.eq:
+                return ConstantVariable(op(left.value, right.value))
+
+            if isinstance(right, ConstantVariable) or isinstance(
+                left, ConstantVariable
+            ):
+                return ConstantVariable(op(left.value, right.value))
```

Comment:
This could still fail if one of the operands does not have `.value` attr, right? That's not guaranteed to be the case here. We have a similar issue with the operator.is implementation (it caused an AttributeError).

Reply:
I think ConstantVariable and StreamVariable are guaranteed to have `.value` attribute. So it should be safe?

Reply:
`operator.eq is not operator.is_` on line 1688?

Reply:
Oh yeah, I guess the invariant holds here by pidgeon hole (one operand must be ConstantVariable).